rest_api/limsdb/queries/run_status.py

```diff
@@ -22,11 +22,13 @@ def to_json(self):
         }
 
 
-def run_status(session):
-    # TODO: make the time threshold a parameter
-    now = datetime.now()
-    threshold = now - timedelta(7)
-    data = queries.runs_info(session, time_since=threshold)
```

Comment:
We need access to the Flask app object here, which means that all other Lims aggregation functions need to have it as an arg, even if it's not used (since we're calling them through a mapping).

If we intend to access the object in those functions too in the future, then that's fair. Alternatively if that's not likely, we could access the object here with `flask.current_app` instead, allowing us to keep the args as they were.